### DISCUSSION TITLE
Sparkline in ui.table

### DISCUSSION THREAD

**Discussion by retsyo** [zauberzeug/nicegui#2052] [Ideas / Feature Requests]
As the title says. For example, [peity vanilla](https://github.com/railsjazz/peity_vanilla) can render sparkline in table's cell. And sorry, I found [peity vanilla](https://github.com/railsjazz/peity_vanilla) on [tabulator's page](https://tabulator.info/examples/5.5#sparklines)

**Comment by falkoschindler** [comment]
Interesting... Here's a proof of concept:
```py
ui.add_head_html('<script src="https://cdn.jsdelivr.net/npm/peity-vanilla/dist/peity-vanilla.min.js"></script>')

columns = [
    {'name': 'name', 'label': 'Name', 'field': 'name', 'align': 'left'},
    {'name': 'age', 'label': 'Age', 'field': 'age'},
    {'name': 'weight', 'label': 'Weight', 'field': 'weight'}
]
rows = [
    {'name': 'Alice', 'age': 18, 'weight': '60, 65, 62, 63, 58'},
    {'name': 'Bob', 'age': 21, 'weight': '70, 75, 80, 73, 82'},
    {'name': 'Carol', 'age': 24, 'weight': '80, 85, 82, 83, 88'},
]
table = ui.table(columns=columns, rows=rows, row_key='name')
table.add_slot('body-cell-weight', '<td :props="props"><span class="line">{{ props.value }}</span></td>')

app.on_connect(lambda: ui.run_javascript('document.querySelectorAll(".line").forEach(e => peity(e, "line"))'))
```

<img width="202" alt="Screenshot 2023-11-21 at 07 52 43" src="https://github.com/zauberzeug/nicegui/assets/5767091/eedaaccb-e1f4-4f27-9cce-d021a59760e7">


But I'm not sure how an integration might look like. The need to call `peity` on the client is a bit inconvenient.

**Comment by Ha44a** [comment]
Re-bumping this as I would find this a useful feature. I think currently you can do it with AG Grid however it requires the enterprise version so quite unattainable
